package.json

```diff
@@ -82,9 +83,13 @@
     "lint:fix": "yarn run lint --fix"
   },
   "lint-staged": {
-    "{packages,examples}/**/*.{js,jsx,ts,tsx,json,css,scss,md}": [
+    "{packages,examples}/**/*.{json,css,scss,md}": [
       "prettier --write",
       "git add"
+    ],
+    "{packages,examples}/**/*.js,jsx,ts,tsx": [
```

Comment:
I split these two file types [per Javiers suggestion](https://github.ibm.com/Watson-Discovery/disco-widgets/issues/166#issuecomment-15076024)

Reply:
I'm going to combine these two file types again, although `eslint` will have no effect on css and scss, this will still invoke prettier which should have an effect